articles/cosmos-db/troubleshoot-java-sdk-v4-sql.md

```diff
@@ -173,12 +173,12 @@ Also add a log4j config.
 ```
 # this is a sample log4j configuration
 
-# Set root logger level to DEBUG and its only appender to A1.
+# Set root logger level to INFO and its only appender to A1.
 log4j.rootLogger=INFO, A1
 
-log4j.category.com.azure.cosmos=DEBUG
-#log4j.category.io.netty=INFO
-#log4j.category.io.projectreactor=INFO
+log4j.category.com.azure.cosmos=INFO
+#log4j.category.io.netty=OFF
```

Comment:
Won't OFF is too extreme? Like not logs at-all. 
Did we consider at-least making it very high level?

Reply:
These are netty and reactor logs, which we don't want. And they are commented out as well. 
These are here to show just in case some one needs to debug netty and reactor (flux / mono / reactor-netty) then they should add these. 
@ktoliver - We are good with the PR to go ahead.